api/walletApi-V0.yml

```diff
@@ -246,6 +239,20 @@ definitions:
         description: error message.
         type: string
         x-nullable: false
+  Operation:
+    type: object
+    required:
+      - operation
+    properties:
+      operation:
+        description: Serialized attributes of the operation to be signed with the key pair corresponding to the given nickname.
+        type: string
+        format: byte
+      batch:
+        description: Whether the sign operation is part of a batch of operations
```

Comment:
More expressive description Example : A boolean property that indicates whether the sign operation is part of a batch of operations. Set to `true` if this operation is part of a batch, otherwise set to `false`.

Reply:
thanks!